components/InputField.tsx

```diff
@@ -29,23 +29,35 @@ const InputField: FC<InputFieldProps> = ({
   max,
   helpMessage,
 }) => {
+  const inputErrorMessageId = `input-${id}-error`
+  const inputHelpMessageId = `input-${id}-help`
+  const inputWapperId = `input-${id}`
```

Comment:
Nitpicking perhaps but this should read `inputWrapperId`. Everything else looks good 👍